### PR TITLE
E2etest concurrency

### PR DESCRIPTION
**Pull request by jaiminpanchal27**
## Type of change

- [x ] Bugfix

## Description of change
End 2 End testing using nightwatch and browserstack was having issues running in parallel. With this fix we are now running only 2 browsers at a time.  To test this following steps must be followed 
- Start tunneling between browserstack and local machine
./BrowserStackLocal --key \<your-browserstack-key\> --force-local 
- 'gulp serve-nw --browserstack' to start the running the tests
- Once finished, execute 'gulp e2etest-report' to view the report in browser.

### CODE REVIEW

File: nightwatch.browserstack.json [hunk 1]
```diff
@@ -38,8 +38,8 @@
         "browserstack.local": true,
         "browserstack.debug": true,
         "browserstack.selenium_version" : "2.53.0",
-        "browserstack.user": "${BROWSERSTACK_USERNAME}",
-        "browserstack.key": "${BROWSERSTACK_KEY}"
+        "browserstack.user": "${BROWSERSTACK_USERNAME1}",
```
Comment (mkendall07) [inline nightwatch.browserstack.json]: Was this intentional change?

File: test/spec/e2e/testcase1/dom-group/allbidders_dom_spec.js [hunk 2]
```diff
@@ -4,9 +4,9 @@ var util = require('../../common/utils.js');
 module.exports = {
   'adequant ad rendering' : function (browser) {
     browser
-      .url('http://localhost:9999/test/spec/e2e/gpt-examples/all_bidders_instant_load.html')
+      .url('http://an.localhost:9999/test/spec/e2e/gpt-examples/all_bidders_instant_load.html')
```
Comment (mkendall07) [inline test/spec/e2e/testcase1/dom-group/allbidders_dom_spec.js]: Should make a note about setting up this in `hosts` file

File: nightwatch.browserstack.json (hunk 1 above)
Comment (jaiminpanchal27) [inline nightwatch.browserstack.json]: nopes. I will update this.

File: test/spec/e2e/testcase1/dom-group/allbidders_dom_spec.js (hunk 2 above)
Comment (jaiminpanchal27) [inline test/spec/e2e/testcase1/dom-group/allbidders_dom_spec.js]: yes would that to instructions.

**Comment by mkendall07** [comment]
LGTM

**Pull request merged**